src/_gettsim/transfers/wohngeld/voraussetzungen.py

```diff
@@ -0,0 +1,259 @@
+    that passed the priority check against Arbeitslosengeld II / Bürgergeld.
+
+    Parameters
+    ----------
+    mindesteinkommen_erreicht_wthh
+        See :func:`mindesteinkommen_erreicht_wthh`.
+    vermögensgrenze_unterschritten_wthh
+        See :func:`vermögensgrenze_unterschritten_wthh`.
+
+    Returns
+    -------
+
+    """
+    return vermögensgrenze_unterschritten_wthh and mindesteinkommen_erreicht_wthh
+
+
+@policy_function()
+def grundsätzlich_anspruchsberechtigt_bg(
+    mindesteinkommen_erreicht_bg: bool,
+    vermögensgrenze_unterschritten_bg: bool,
+) -> bool:
+    """Check whether the household meets the conditions for Wohngeld.
+
+    This target is used for the priority check calculation against Arbeitslosengeld II /
+    Bürgergeld on the Bedarfsgemeinschaft level.
+
+    Parameters
+    ----------
+    mindesteinkommen_erreicht_bg
+        See :func:`mindesteinkommen_erreicht_bg`.
+    vermögensgrenze_unterschritten_bg
+        See :func:`vermögensgrenze_unterschritten_bg`.
+
+    Returns
+    -------
+
+    """
+    return mindesteinkommen_erreicht_bg and vermögensgrenze_unterschritten_bg
+
+
+@policy_function()
+def vermögensgrenze_unterschritten_wthh(
+    demographics__vermögen_wthh: float,
+    demographics__anzahl_personen_wthh: int,
+    wohngeld_params: dict,
+) -> bool:
+    """Wealth is below the eligibility threshold for housing benefits.
+
+    Parameters
+    ----------
+    demographics__vermögen_wthh
+        See :func:`demographics__vermögen_wthh <demographics__vermögen_wthh>`.
+    demographics__anzahl_personen_wthh
+        See :func:`demographics__anzahl_personen_wthh`.
+    wohngeld_params
+        See params documentation :ref:`wohngeld_params <wohngeld_params>`.
+
+    Returns
+    -------
+
+    """
+
+    return vermögensprüfung_formel(
+        vermögen=demographics__vermögen_wthh,
+        anzahl_personen=demographics__anzahl_personen_wthh,
+        params=wohngeld_params,
+    )
+
+
+@policy_function()
+def vermögensgrenze_unterschritten_bg(
+    demographics__vermögen_bg: float,
+    demographics__anzahl_personen_bg: int,
+    wohngeld_params: dict,
+) -> bool:
+    """Wealth is below the eligibility threshold for housing benefits.
+
+    Parameters
+    ----------
+    demographics__vermögen_bg
+        See :func:`demographics__vermögen_bg <demographics__vermögen_bg>`.
+    demographics__anzahl_personen_bg
+        See :func:`demographics__anzahl_personen_bg`.
+    wohngeld_params
+        See params documentation :ref:`wohngeld_params <wohngeld_params>`.
+
+    Returns
+    -------
+
+    """
+
+    return vermögensprüfung_formel(
+        vermögen=demographics__vermögen_bg,
+        anzahl_personen=demographics__anzahl_personen_bg,
+        params=wohngeld_params,
+    )
+
+
+@policy_function()
+def mindesteinkommen_erreicht_wthh(
+    arbeitslosengeld_2__regelbedarf_m_wthh: float,
+    einkommen_für_mindesteinkommen_check_m_wthh: float,
```

Comment:
```suggestion
    einkommen_für_mindesteinkommen_m_wthh: float,
```
Clear enough and we do not use that terminology for the function in question.